docs/reference/changelog-r2019.md

```diff
@@ -17,6 +17,7 @@ Released on XXX YYth, 2019.
     - Added a `village_center` world.
   - Enhancements
     - Improved the intensity and color of the bus, truck and car vehicle lights.
+    - macOS and Linux: Use POSIX shared memory segments to fix the limit problems on macOS and allow for snap packaging with strict confinement on Linux.
```

Comment:
I'm wondering if mentioning here `snap` is useful, it seems to be unrelated.

Reply:
It's related. Without POSIX shared memory, there is no way to create a strictly confined snap.